src/Provider/ChannelColorProviderInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Provider;
+
+interface ChannelColorProviderInterface
+{
+    public function __invoke(string $channelCode): string;
```

Comment:
It is just my personal opinion, and I'm not sure about standards on this plugin, but I found it cleaner to name functions if they are part of the interface. IMHO `__invoke` is perfect but for controllers, command handlers and this type of classes (without interfaces). Even simple:

```suggestion
    public function provide(string $channelCode): string;
```

Reply:
I get your point but what can a *Provider do instead of... providing? I can see both pros and cons of using __invoke() but to me, channelColorProvider->provide() sounds a bit redundant. WDYT?

Reply:
That's correct, and maybe the problem is just with my aesthetic feeling or that I'm used to these redundant functions. On the other hand, `__invoke` is a magic method and is meant to be used like: `$class($arg)`. When we are using it with `$class->__invoke($arg)` is the same as we would agree for example for `run` function. It does not provide any benefit and looks bad for me. 

IMHO lines like https://github.com/Sylius/InvoicingPlugin/pull/124/files#diff-94b24b9abcb05ca0be18c811d5d20a32R24 is a trigger for wrong `__invoke` usage. In such a case I prefer redundant functions, so at least we don't have a call for a function with two dashes at the beginning.